core/src/main/scala/kafka/server/BrokerServer.scala

```diff
@@ -394,7 +394,7 @@ class BrokerServer(
 
       val shareFetchSessionCache : ShareSessionCache = new ShareSessionCache(config.shareGroupMaxGroups * config.shareGroupMaxSize,
         KafkaServer.MIN_INCREMENTAL_FETCH_SESSION_EVICTION_MS)
-      val sharePartitionManager = new SharePartitionManager(replicaManager, Time.SYSTEM, shareFetchSessionCache)
+      val sharePartitionManager = new SharePartitionManager(replicaManager, Time.SYSTEM, shareFetchSessionCache, config.shareGroupRecordLockDurationMs)
```

Comment:
Where do we use `config.shareGroupMaxRecordLockDurationMs`?

Reply:
It is only used to compare `group.share.record.lock.duration.ms` and `group.share.record.lock.max.duration.ms` [here](https://github.com/confluentinc/kafka/pull/1151/files#diff-cbe6a8b71b05ed22cf09d97591225b588e9fca6caaf95d3b34a43262cfd23aa6R2387)